### ISSUE TITLE
App Crashes on Android

### ISSUE THREAD

**Issue by tannerstern** [HitPauseApp/HitPause#31]
# Bug Report 

## App Breaking? (Causes crashes, screens full of red text, etc.)
Yes

## Frequency (Only on Expo Start / Every reload of the app / every couple reloads / randomly) 
Every reload of the app, instantaneously

### Describe Bug
Opening the project on an Android device (through Expo) causes the app to immediately crash without any error messages. When opening the project, the splash screen is displayed, and the login screen appears for a split second before the (Expo) app crashes and closes completely.

Using Android's bug reporting feature, I was able to get a little bit more information on what's going on. That screenshot is included below.

#### Did this happen after you installed/modified a package? If so, which one. 
Not that I'm aware of.

### Supporting Images, Code Blocks, Etc. 
![image](https://user-images.githubusercontent.com/43083093/95821920-eb0f4000-0cf8-11eb-81cc-9ad2fd6220b5.png)

**Comment by tannerstern** [comment]
@drewweaver27 your comment on #26 points to an issue with the authentication flow. I was able to get the "Login" page to load on Appetize as you did, and it was only after entering my credentials that the app crashed. Subsequent attempts to open the app lead to immediate crashes, which suggests that the authentication succeeded (?) and the app is trying to log in with the credentials previously supplied.

**Comment by drewweaver27** [comment]
@tannerstern Hmm perhaps the navigation changes are causing an issue? Just logged out on my local environment to go through the auth flow again and noticed this error popped up on the login screen which is persistent after refreshing multiple times  

<img width="1440" alt="Screen Shot 2020-10-13 at 3 38 59 PM" src="https://user-images.githubusercontent.com/24489555/95907888-3e6da680-0d6a-11eb-87f3-7a3c6efdc2b8.png">


Then on both screens this warning is there 

<img width="1440" alt="Screen Shot 2020-10-13 at 3 36 42 PM" src="https://user-images.githubusercontent.com/24489555/95907795-1d0cba80-0d6a-11eb-943d-911be83a1d25.png">

**Comment by drewweaver27** [comment]
@tannerstern 

```<NavigationContainer ref={containerRef}>
              {/* Display authentication screens or app screens based on userToken */}
              { authUser == null ? (
                <Stack.Navigator initialRouteName={preAuthNavState} headerMode="none">
                  <Stack.Screen name="Login" component={Login} />
                  <Stack.Screen name="SignUp" component={SignUp} />
                  <Stack.Screen name="ResetPassword" component={ResetPassword} />
                  <Stack.Screen name = "JournalEntry" component = {JournalEntry}/>
                </Stack.Navigator>
              ) : (
```

Is the Journal Entry supposed to be on the authentication stack?

**Comment by Willnew98** [comment]
I feel like the Journal Page should be, but the JournalEntry Page should not.

**Comment by tannerstern** [comment]
The same fixes that worked for #37 also seem to have stabilized the Android build. This all thanks to [Drew's big bugfix commit](https://github.com/HitPauseApp/HitPause/commit/74e8d5dfc903efb17796e28cb11eb5ec97401608).
